packages/flutter/lib/src/widgets/tap_region.dart

```diff
@@ -635,5 +635,6 @@ class TextFieldTapRegion extends TapRegion {
     super.onTapInside,
     super.consumeOutsideTaps,
     super.debugLabel,
-  }) : super(groupId: EditableText);
+    super.groupId = EditableText,
+  }) : super();
```

Comment:
Since we now pass everything with `super` parameters, isn't the `super();` redundant here?

Reply:
Yes, your suggestion is very good, and the unnecessary code has been removed.

Reply:
@navaronbracke Are there any other problem?